.circleci/config.yml

```diff
@@ -37,11 +37,15 @@ jobs:
             echo 'export PATH=$HOME/.local/bin:$PATH' >> $BASH_ENV
             curl -OL https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip
             unzip protoc-$PROTOC_VERSION-linux-x86_64.zip -d $HOME/.local
+            rm protoc-$PROTOC_VERSION-linux-x86_64.zip
             sudo apt install -y cmake golang
       - run:
           name: Test Rust
           command: make rust
           no_output_timeout: 1800s
+      - run:
+          name: Check if all files are regenerated
+          command: git update-index --refresh &>/dev/null || (git diff; echo please "make rust" before submitting a PR; exit 1)
```

Comment:
IIRC, `file_descriptor_proto_data` is non-deterministic and can change when built on different platforms (I'm not sure exactly when it changes, but I have seen it change often when rebuilding the protos). Seems like that would trigger this check?

Reply:
I don't think such field exist anymore after lite-runtime is enabled.